core/templates/domain/exploration/editable-exploration-backend-api.service.ts

```diff
@@ -43,9 +43,10 @@ export class EditableExplorationBackendApiService {
       this._getExplorationUrl(explorationId, applyDraft)).toPromise();
   }
 
+  // The version number is null, whenever we create a deafult exploration.
```

Comment:
Can we somehow get rid of this `null` and start with some number?

Reply:
Done, now if the data version is not provided we are assigning 0 to it instead of null.

Reply:
0 is not a correct version number

Reply:
Updated now to 1. Thanks!